events/events.go

```diff
@@ -283,26 +284,26 @@ func emitUserEvent(help string, failure bool, details pb.IsEventDetails) {
 	}
 }
 
-func ToMalfeasancePB(nodeID types.NodeID, mp *types.MalfeasanceProof, includeProof bool) *pb.MalfeasanceProof {
+func ToMalfeasancePB(nodeID types.NodeID, mp *wire.MalfeasanceProof, includeProof bool) *pb.MalfeasanceProof {
```

Comment:
This probably also needs to be in the `malfeasance` package and call handlers in other packages (via interfaces) to convert a typed malfeasance proof into the GRPC type that we then return via the event stream.